src/commands/transaction/update.ts

```diff
@@ -79,18 +86,23 @@ const displayUpdateResult = (response: SuccessResponse) => {
   console.log(chalk.cyan(chalk.bold(`Transaction ID ${oldTransaction.id}`)));
 
   const table = display.table.createCompact();
-  pushUpdateResultRow(
-    "Title:",
-    table,
-    oldTransaction.title,
-    newTransaction.title
-  );
-  pushUpdateResultRow(
-    "Change in amount:",
-    table,
-    oldTransaction.amountChange,
-    newTransaction.amountChange
-  );
+  if (oldTransaction.title === newTransaction.title && oldTransaction.amountChange === newTransaction.amountChange) {
```

Comment:
I'd much prefer not having this big if statement with other if statements nested in it for this particular requirement. From my angle, it looks like the cost of code readability is larger than the benefit of having this new feature. Can you think of another way of doing this?

Reply:
I think a good way is to break them down into two cases, one of which pushes a row into the table for the title, one of which for amountChange, then if neither is executed then the table is empty and displays a warning message using chalk.